src/Microsoft.Kusto.ServiceLayer/HostLoader.cs

```diff
@@ -52,7 +52,7 @@ internal static ServiceHost CreateAndStartServiceHost(SqlToolsContext sqlToolsCo
                     // Start the service only after all request handlers are setup. This is vital
                     // as otherwise the Initialize event can be lost - it's processed and discarded before the handler
                     // is hooked up to receive the message
-                    serviceHost.Start().Wait();
+                    serviceHost.Start().GetAwaiter().GetResult();
```

Comment:
Was this something flagged by the SDK update? 

And what exactly is this change doing? Looking around it seems like it just makes the stack trace a bit cleaner - is that it or is there some other benefit doing it this way brings?

Reply:
It's not flagged, but as a practice this should be followed, the main benefit of `GetAwaiter().GetResult()` is is preseves exception stacktrace and doesn't throw `AggregateException` as `Wait()` or `.Result` do.

Reply:
Thanks for the context. Is there some linting/style rule we can enable to have this show up as a warning? Definitely sounds useful but I don't think most people will know about or remember to do this unless we have the build remind us.